src/grid/Col.js

```diff
@@ -1,8 +1,13 @@
+/*eslint-disable no-console */
 import PropTypes from 'prop-types';
 import React from 'react';
 import { View, StyleSheet, TouchableOpacity } from 'react-native';
 
 const Col = props => {
+  console.warn(
+    'Warning: Col has been deprecated and will be removed in a future version of React Native Elements'
```

Comment:
@binoy14 nope. Imo, Grid is just a beginner friendly version of using flex. I think a beginner/junior dev is better off learning flex as it will help them in the long run. Hence, we are going to be removing it entirely from RNE.